src/uu/uname/src/uname.rs

```diff
@@ -15,92 +15,156 @@ use uucore::{
 const ABOUT: &str = help_about!("uname.md");
 const USAGE: &str = help_usage!("uname.md");
 
-pub mod options {
-    pub static ALL: &str = "all";
-    pub static KERNEL_NAME: &str = "kernel-name";
-    pub static NODENAME: &str = "nodename";
-    pub static KERNEL_VERSION: &str = "kernel-version";
-    pub static KERNEL_RELEASE: &str = "kernel-release";
-    pub static MACHINE: &str = "machine";
-    pub static PROCESSOR: &str = "processor";
-    pub static HARDWARE_PLATFORM: &str = "hardware-platform";
-    pub static OS: &str = "operating-system";
+static ALL: &str = "all";
+static KERNEL_NAME: &str = "kernel-name";
+static NODENAME: &str = "nodename";
+static KERNEL_VERSION: &str = "kernel-version";
+static KERNEL_RELEASE: &str = "kernel-release";
+static MACHINE: &str = "machine";
+static PROCESSOR: &str = "processor";
+static HARDWARE_PLATFORM: &str = "hardware-platform";
+static OS: &str = "operating-system";
```

Comment:
I'd like to keep this for consistency with the other utilities. It's also a change that is not really relevant to nushell.

Reply:
That is still there,   what I changed was `mod Options {}` to the same consistent way I saw in `mv` and `cp` with `pub struct Options {}` unless you meant something else.?

Reply:
You did removed the `options` module around the arguments right? That's what I meant

Reply:
ah ok sorry, the `+` liens in the diff tripped me up, bad brain fart , but added it back again